src/components/ui/Input/index.stories.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import Input from '.';
+
+// export default {
+//   component: Input,
+//   title: 'components/ui/Input',
+// };
+
+// export const largeLoader = () => (
+//   <Input size="large" />
+// );
```

Comment:
I think you forgot to clean this

Reply:
Thanks for the catch. I was going to write the story for that.